app_dart/lib/src/request_handlers/update_branches.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:cocoon_service/src/model/appengine/branch.dart' as md;
+import 'package:gcloud/db.dart';
+import 'package:github/github.dart' show Branch, GitHub, RepositoryCommit, RepositorySlug;
+import 'package:process_runner/process_runner.dart';
```

Comment:
We should use `package:process`. We don't need the extra complexity of process_runner, and it's already in our dep tree

Reply:
It might not be ideal to import dart:io, but it seems like process/process.dart itself wasn't quite enough. I looked at the examples at https://github.com/google/process.dart/blob/master/test/src/interface/common_test.dart and imported dart:io to get process to work. but for testing the mock of a Process class, i.e., a mockProcess would not have a valid mockProcess.run function because the run function is abstract. I am not sure if there is a valid way for us to test process and provide a fake result in tests? Thank you!

Reply:
Here's an example of testing with `package:process`: https://github.com/flutter/cocoon/blob/main/device_doctor/test/src/host_utils_test.dart

Reply:
Thanks a lot! imported dart:io and updated accordingly.